certified-connectors/OzyApprovals/apiDefinition.swagger.json

```diff
@@ -1 +1,1285 @@
+                    },
+                    "approval_stages_under": {
+                      "type": "string",
+                      "description": "No of stages under this process",
+                      "title": ""
+                    },
+                    "approval_process_stages": {
+                      "type": "array",
+                      "items": {
+                        "type": "object",
+                        "properties": {
+                          "id": {
+                            "type": "string",
+                            "description": "Stage ID"
+                          },
+                          "names": {
+                            "type": "string",
+                            "description": "Stage Names"
+                          },
+                          "description": {
+                            "type": "string",
+                            "description": "Stage Description"
+                          },
+                          "deleted": {
+                            "type": "string",
+                            "description": "Is stage deleted?"
+                          }
+                        }
+                      }
+                    },
+                    "msg": {
+                      "type": "string",
+                      "description": "msg"
+                    }
+                  },
+                  "description": "msg"
+                }
+              }
+            },
+            "headers": {
+              "Content-Type": {
+                "description": "Content-Type",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/get_approval_processes"
+          },
+          {
+            "required": true,
+            "name": "query",
+            "in": "body",
+            "schema": {
+              "x-ms-dynamic-properties": {
+                "itemValuePath": "msg",
+                "parameters": {
+                  "process_id": {
+                    "parameterReference": "process_id"
+                  }
+                },
+                "operationId": "GetProcessFields"
+              },
+              "x-ms-dynamic-schema": {
+                "parameters": {
+                  "process_id": {
+                    "parameter": "process_id"
+                  }
+                },
+                "operationId": "GetProcessFields",
+                "value-path": "msg"
+              }
+            },
+            "x-ms-visibility": "important",
+            "description": "Register New Approval Request"
+          }
+        ],
+        "operationId": "RegisterNewProcessRequest",
+        "description": "Start a new approval process that can undergo multiple approval stages."
+      }
+    },
+    "/activate_approval_stage": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
```

Comment:
Please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.